docs/installation.adoc

```diff
@@ -0,0 +1,55 @@
+= Installing RNP
+
+Binaries that will be installed:
+
+* `rnp`
+* `rnpkeys`
+
+== On macOS using Homebrew
+
+[source,console]
+----
+brew tap riboseinc/rnp
+brew install rnp
+----
+
+== On RHEL and CentOS via YUM
+
+[source,console]
+----
+rpm --import https://github.com/riboseinc/yum/raw/master/ribose-packages.pub
+curl -L https://github.com/riboseinc/yum/raw/master/ribose.repo > /etc/yum.repos.d/ribose.repo
+yum install -y rnp
+----
+
+== On Debian
+
+[source,console]
+----
+# Clone the repository by version tag (or omit it to get the latest sources)
+sudo apt install git
+git clone https://github.com/riboseinc/rnp.git -b v0.11.0
```

Comment:
We have an 0.12.0 release -- I wonder if we should try to keep this line up-to-date?

Reply:
We should definitely keep installation steps up to date, but I wonder if this one in particular we can use some sort of variable...?

Reply:
How about adding this item to release checklist @dewyatt 
Alongside any changelog updates and such

Reply:
@dewyatt if no objections, I’ll add this to the release workflow doc (around where it tells the maintainer to update CHANGELOG)

Reply:
@ronaldtse I'm not aware of any at the moment. @strogonoff That sounds like a good idea to me.